src/sentry/processing/realtime_metrics/redis.py

```diff
@@ -132,20 +131,20 @@ def get_counts_for_project(self, project_id: int) -> Iterable[base.BucketedCount
         This may throw an exception if there is some sort of issue fetching counts from the redis
         store.
         """
-        key_prefix = f"{self._counter_key_prefix()}:{project_id}:"
+        bucket_size = self._counter_bucket_size
+        now = int(time.time())
+        now_timestamp = now - now % bucket_size
```

Comment:
it looks like the only place this is being used performs an additional operation by rolling the value up to the end of bucket it occupies in L141 (`now_timestamp + bucket_size`). should the rollup just be performed here instead?

Reply:
Tbh I think it's clearer the way it is now.